roles/process_nmstate/templates/nmstate.yml.j2

```diff
@@ -10,6 +10,9 @@ dns-resolver:
 interfaces:
 {% for interface in network_config.interfaces %}
   - name: {{ interface.name }}
+    {% if interface.mac is defined %}
+    mac: {{ interface.mac }}
```

Comment:
I think its `mac-address` in NMState :)

Reply:
Fixed, I've mainly included it because I've seen several network configs making use of MAC addresses, but the template for nmstate didn't have it.

Reply:
Yeah we use it to populate `mac_interface_map` which the installers use to select the correct nmstate for the host.